versions/3.0.md

```diff
@@ -106,20 +106,19 @@ For example, if a field has an array value, the JSON array representation will b
    "field": [...]
 }
 ```
-
-While the API is described using JSON, input to and output from the API is not required to be JSON.
-
 All field names in the specification are **case sensitive**.
 
 The schema exposes two types of fields: Fixed fields, which have a declared name, and Patterned fields, which declare a regex pattern for the field name.
 
-Multiple occurrences of patterned fields need to have unique names. 
+Patterned fields MUST have unique names within the containing object. 
 
 In order to preserve the ability to round-trip between YAML and JSON formats, YAML version [1.2](http://www.yaml.org/spec/1.2/spec.html) is recommended along with some additional constraints:
 
 - Tags MUST be limited to those allowed by the [JSON Schema ruleset](http://www.yaml.org/spec/1.2/spec.html#id2803231).
 - Keys used in YAML maps MUST be limited to a scalar string, as defined by the [YAML Failsafe schema ruleset](http://yaml.org/spec/1.2/spec.html#id2802346).
 
+**Note:** While APIs are described with JSON or YAML Open API documents, the API request and response bodies and other content are not required to be JSON or YAML.
```

Comment:
Re: `described with JSON or YAML Open API documents`, may I suggest `described by OpenAPI documents in YAML or JSON format`?